Amazing Codemirror UI

serzero2007 opened the pull request:
[![Review](https://codenvy.com/factory/resources/codenvy-review.svg)](https://codenvy.com/f?id=a9phxofmevw6k1a8)

I have implemented the codemirror support

serzero2007 committed:
UNSTABLE-CODEMIRROR-SUPPORT: Amazing Codemirror UI

serzero2007 committed:
library added

required library added

zohooo commented:
Thanks. But there are some problems:

  1. It's good to update codemirror to latest version, but please put files into `codemirror` folder  (and override the old version) rather than into `codemirror-5.3` folder. Also unnecessary subfolders `bin`, `demo`, `doc` and `test`, etc should not be included.
  2. The content of `index.html` is totally replaced, and I could not see what's the change. Please obey the indent style in this project.

serzero2007 committed:
Many fixes and improvements

UI code refactoring.
Buttons deleted, Codemirror finally added as it meant to be.
Dialogs UI code deleted.
SkyDrive code deleted.
Much of removed functionality is to be restored in future.

serzero2007 committed:
Minor Update
